### ISSUE TITLE
How to setup isolation level?

### ISSUE THREAD

**Issue by melinhunter** [OpenInformix/IfxPy#27]
info = IfxPy.server_info (conn)
info.DFT_ISOLATION is CS.
info.ISOLATION_OPTION is ('UR', 'CS', 'RR')

how to do change default isolation level to read uncommitted?

**Comment by jsagrera** [comment]
I don't think we have any option to change the isolation within the driver itself, so the only way will be to do it directly with an SQL statement: 

    IfxPy.exec_immediate(conn, "SET ISOLATION DIRTY READ")
